generators/generator-base-private.js

```diff
@@ -1413,4 +1414,77 @@ module.exports = class extends Generator {
             );
         }
     }
+
+    /**
+     * Formats source code so for compare
+     * @param {string} sourceCode - source code to format
+     * @param {string} extension - extension for configuring Prettier properly, for example `.ts` for Typescript files
+     */
+    formatSourceCodeForCompare(sourceCode, extension) {
+        let options = {};
+        if (extension === '.ts') {
+            options = { parser: 'typescript' };
+        }
+        try {
+            // normalize line endings, because for example in Windows this can be in generated content \n and in disk \r\n
+            return prettierFormat(sourceCode, options)
+                .replace(new RegExp('\\r\\n', 'g'), '\n')
+                .replace(new RegExp('\\r', 'g'), '\n');
+        } catch (e) {
+            return sourceCode.replace(new RegExp('\\s', 'g'), '');
```

Comment:
why don't you use this form? `replace(/WHATEVER/g, '\n')`

Reply:
The `new RegExp` command was just first one that came to my mind. Your suggestion is of course cleaner, shorter and better readable. I made this change. Thanks!